__tests__/app.test.js

```diff
@@ -249,3 +249,53 @@ describe("POST api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH api/articles/article_id", () => {
+  test("200: returns article with updated votes", () => {
+    const newVotes = {
+      inc_votes: 10,
+    };
+    return supertest(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        expect(article).toHaveProperty("article_id", expect.any(Number));
+        expect(article).toHaveProperty("title", expect.any(String));
+        expect(article).toHaveProperty("topic", expect.any(String));
+        expect(article).toHaveProperty("author", expect.any(String));
+        expect(article).toHaveProperty("body", expect.any(String));
+        expect(article).toHaveProperty("created_at", expect.any(String));
+        expect(article).toHaveProperty("votes", expect.any(Number));
+        expect(article).toHaveProperty("article_img_url", expect.any(String));
+      });
+  });
+  test("404: returns appropriate response when sending a patch request with an invalid article_id", () => {
+    const newVotes = {
+      inc_votes: 10,
+    };
+    return supertest(app)
+      .patch("/api/articles/999")
+      .send(newVotes)
+      .expect(404)
+      .then((res) => {
+        const { msg } = res.body;
+        expect(msg).toBe("Not found");
+      });
+  });
+  test("400: returns appropriate response when sending a patch with  an invalid article_id", () => {
```

Comment:
Great error tests so far! 
What if the votes body was incorrect in some way? e.g. inc_votes property is not a number